### PR TITLE
Drop Python 2.7 support, add support for Python 3

### PR DESCRIPTION
**Pull request by pnuu**
This PR converts the code to work on Python 3, and at the same time drops Python 2 support.

### CODE REVIEW

**Comment by pnuu** [comment]
The conversion was done with `2to3.py`, so there might be something that it didn't catch.

File: trollcast/client.py [hunk 1]
```diff
@@ -83,11 +83,11 @@ def __init__(self, addresses, translate=False):
 
     @property
     def sub_addr(self):
-        return dict(zip(self.subscribers, self._addresses))
+        return dict(list(zip(self.subscribers, self._addresses)))
```
Comment (mraspaud) [inline trollcast/client.py]:
```suggestion
        return dict(zip(self.subscribers, self._addresses))
```

File: trollcast/client.py [hunk 2]
```diff
@@ -83,11 +83,11 @@ def __init__(self, addresses, translate=False):
...
 
     @property
     def addr_sub(self):
-        return dict(zip(self._addresses, self.subscribers))
+        return dict(list(zip(self._addresses, self.subscribers)))
```
Comment (mraspaud) [inline trollcast/client.py]:
```suggestion
        return dict(zip(self._addresses, self.subscribers))
```

File: trollcast/client.py [hunk 3]
```diff
@@ -525,7 +525,7 @@ def run(self):
                              ": " + str(message))
                 self._hb[str(sender)].reset()
 
-                for addr, req in self._requesters.items():
+                for addr, req in list(self._requesters.items()):
```
Comment (mraspaud) [inline trollcast/client.py]:
```suggestion
                for addr, req in self._requesters.items():
```

File: trollcast/client.py [hunk 4]
```diff
@@ -546,7 +546,7 @@ def stop(self):
         """Stop buffering.
         """
         self._sub.stop()
-        for timer in self._hb.values():
+        for timer in list(self._hb.values()):
```
Comment (mraspaud) [inline trollcast/client.py]:
```suggestion
        for timer in self._hb.values():
```

File: trollcast/client.py [hunk 5]
```diff
@@ -659,7 +659,7 @@ def get_all(self, satellites):
 
                 except Empty:
                     pass
-                for sat, (utctime, elevation) in sat_last_seen.items():
+                for sat, (utctime, elevation) in list(sat_last_seen.items()):
```
Comment (mraspaud) [inline trollcast/client.py]:
```suggestion
                for sat, (utctime, elevation) in sat_last_seen.items():
```

File: trollcast/client.py [hunk 6]
```diff
@@ -726,7 +726,7 @@ def get_all(self, satellites):
                             del sat_last_seen[sat]
                             first_time = None
         except KeyboardInterrupt:
-            for sat, (utctime, elevation) in sat_last_seen.items():
+            for sat, (utctime, elevation) in list(sat_last_seen.items()):
```
Comment (mraspaud) [inline trollcast/client.py]:
```suggestion
            for sat, (utctime, elevation) in sat_last_seen.items():
```

File: trollcast/client.py [hunk 7]
```diff
@@ -875,5 +875,5 @@ def send_lineinfo_to_server(self, *args, **kwargs):
     def stop(self):
         HaveBuffer.stop(self)
         self.loop = False
-        for req in self._requesters.values():
+        for req in list(self._requesters.values()):
```
Comment (mraspaud) [inline trollcast/client.py]:
```suggestion
        for req in self._requesters.values():
```

File: trollcast/oldserver.py [hunk 8]
```diff
@@ -440,7 +440,7 @@ def __init__(self, holder, configfile, *args, **kwargs):
 
     def __del__(self, *args, **kwargs):
         self._socket.close()
-        for mirror in self.mirrors.values():
+        for mirror in list(self.mirrors.values()):
```
Comment (mraspaud) [inline trollcast/oldserver.py]:
```suggestion
        for mirror in self.mirrors.values():
```

File: trollcast/server.py [hunk 9]
```diff
@@ -801,7 +802,7 @@ def run(self):
             mirror_next_pass = self._schedule_reader.mirror_next_pass
             passes = []
             if next_pass is not None:
-                passes.append(dict(zip(keys, next_pass)))
+                passes.append(dict(list(zip(keys, next_pass))))
```
Comment (mraspaud) [inline trollcast/server.py]:
```suggestion
                passes.append(dict(zip(keys, next_pass)))
```

**Review by mraspaud — CHANGES_REQUESTED** [review]
Thanks for the conversion

**Comment by pnuu** [comment]
@mraspaud thanks for going through the code! I didn't realize `2to3.py` would apply `list()` everywhere where dicts are iterated. Travis build was triggered for every suggestion, so it got a bit confused, but is now running the tests.

**Review by mraspaud — APPROVED** [review]

**Pull request merged**